core/chains/evm/txmgr/eth_resender.go

```diff
@@ -33,11 +32,11 @@ const unconfirmedTxAlertLogFrequency = 2 * time.Minute
 // Previously we relied on the bumper to do this for us implicitly but there
 // can occasionally be problems with this (e.g. abnormally long block times, or
 // if gas bumping is disabled)
-type EthResender[ADDR types.Hashable[ADDR], TX_HASH types.Hashable[TX_HASH], BLOCK_HASH types.Hashable[BLOCK_HASH]] struct {
-	txStore             txmgrtypes.TxStore[ADDR, big.Int, TX_HASH, BLOCK_HASH, NewTx[ADDR], *evmtypes.Receipt, EthTx[ADDR, TX_HASH], EthTxAttempt[ADDR, TX_HASH], int64, int64]
+type EthResender[CHAIN_ID txmgrtypes.Id, ADDR types.Hashable[ADDR], TX_HASH types.Hashable[TX_HASH], BLOCK_HASH types.Hashable[BLOCK_HASH], SEQ txmgrtypes.Sequence] struct {
```

Comment:
Maybe out of scope for this PR, but have we considered favoring the 'local' `package types` when naming these? e.g.:
```suggestion
type EthResender[CHAIN_ID types.Id, ADDR common.Hashable[ADDR], TX_HASH common.Hashable[TX_HASH], BLOCK_HASH common.Hashable[BLOCK_HASH], SEQ types.Sequence] struct {
```
Excess packages and generic package names are usually anti-patterns in Go because of these sorts of naming collisions.

Reply:
The chains/evm/txmgr and common/txmgr packages will merge soon, when we finish this refactoring, so this issue will be moot then.